Testscripts/Linux/prepare_ltpt_vm.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+set -x
+
+trap '' HUP
+
+HOMEDIR=$(pwd)
+# Source utils.sh
+. utils.sh || {
+	echo "ERROR: unable to source utils.sh!"
+	echo "TestAborted" > state.txt
+	exit 0
+}
+
+NTTTCP_REPO="https://github.com/Microsoft/ntttcp-for-linux"
+RAID_SETUP_SCRIPT="./CreateRaid.sh"
+
+# Source constants file and initialize most common variables
+UtilsInit
+
+function append_date {
+    while read line
+    do
+        echo "[$(date +%Y-%m-%d-%S)] $line"
+    done < "${1:-/dev/stdin}"
+}
+
+function build_ntttcp {
+    if [[ ! $NTTTCP_REPO ]];then
```

Comment:
You can use existed function build_ntttcp in utils.sh

Reply:
removed local function